client/src/Tickets/TicketModal.jsx

```diff
@@ -84,26 +86,78 @@ function Charger({ stateChange }) {
   );
 }
 
-const AddTicketModel = ({ update, data, stateChange, toggleUserModal, setToggleUserModal }) => {
+const TicketModal = ({ onCreate, onUpdate }) => {
+  const navigate = useNavigate();
+  const { ticketId } = useParams();
```

Comment:
By switching to use ROUTES to manage the modal state, we can pull the id of the ticket being edited from the URL

Reply:
I couldn't remember how to get the id to appear in the url and I knew it was in the code base we worked on in our first half. Now I can see it can be achieved with the routes tag and navigate().